meson.build

```diff
@@ -62,11 +61,6 @@ meson.add_dist_script(
   meson.source_root()
 )
 
-cargo_sources = files(
-  'Cargo.lock',
-  'Cargo.toml',
-)
-
```

Comment:
Even if already closed, this should perhaps be pointed out in https://gitlab.gnome.org/World/Rust/gtk-rust-template/-/merge_requests/70. From my research, it seems to be unused.

Reply:
Yep, I'm going to create a MR in there.